### ISSUE TITLE
Add "Compact layout" option

### ISSUE THREAD

**Issue by vlodko** [swiety85/angular2gridster#93]
If you've heard about [Packery ](https://packery.metafizzy.co/) or similar project, they feature called layout packing - algorithm puts items together removing any gaps between them.

Please, see demo here: [https://tiberiuzuld.github.io/angular-gridster2/angular/]
There is an option called Compact type.

**Do you have any plan regarding this feature?**

It would very useful in our current project.
Thanks.

**Comment by swiety85** [comment]
Hi, "Compact type" option (top/left/...) in angular-gridster2 is by default turned on in angular2gridster - it's called "direction" (vertical/horizontal). It's like gravity to top or to left (or both).
When it comes to layout packing algorithm in Packery, I think it's something different - like your wrote - it's removing any gaps between items. This is more complex feature. I will add your suggestion to project road map, because it's good idea, but I can not say for now when it will be implemented.
If you have any other questions or suggestions, feel free to ask.

**Comment by vlodko** [comment]
It would be great to have ETA for that feature.
Thanks for quick reply!

**Comment by swiety85** [comment]
I can try to do it till the end of this year.

**Comment by vlodko** [comment]
Sounds reasonable, thank you @swiety85

**Comment by swiety85** [comment]
Hi, unfortunately I will not be able to deliver this feature till the end of this year. I have many urgent things to do and it's hard to say when exactly this feature will be delivered.

**Comment by vlodko** [comment]
@swiety85 sorry to hear that. But thanks for response anyway.
One more question: are you going to support all four edges for resizing card?

**Comment by swiety85** [comment]
Yes, angular2gridster supports resizing in all direction. There is a small bug when resizing in north direction, but will be fixed soon. Besides that you can control in which direction resize will be possible by following [gridster option](https://github.com/swiety85/angular2gridster/wiki/API-Documentation#inputs):
```js
resizeHandles: {
      s: true,
      e: true,
      n: true,
      w: true,
      se: true,
      ne: true,
      sw: true,
      nw: true
    }
```
About this layout packing - if you would have this algorithm, I'm sure it's quite ease to apply it on items configuration in your code, outside of the gridster component.

**Comment by vlodko** [comment]
@swiety85 that's nice. But I would update the demo page to all possible options.
